scripts/ae_bundle_streamlines.py

```diff
@@ -0,0 +1,438 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script for the inference of FINTA
```

Comment:
Does not correspond to what the script does.

Reply:
Done